<launch>
    <node name="load_pcl" pkg="pcl_tools" type="load_pcl" output="screen" >
        <param name="pcl2_output" type="string" value="/loaded_pcl2" />
        <param name="pcl_filepath" type="string" value="$(find pcl_tools)/doc/examples/load/test.ply" />
        <param name="include_tf" type="bool" value="true" />
            <param name="tf_filepath" type="string" value="$(find pcl_tools)/doc/examples/load/test_tf.pose" />
            <param name="point_cloud_frame_id" type="string" value="/camera" />
            <param name="world_frame_id" type="string" value="/world" />
    </node>
    <node name="save_pcl" pkg="pcl_tools" type="save_pcl" output="screen" >
        <param name="pcl2_input" type="string" value="/loaded_pcl2" />
        <param name="pcl_filepath" type="string" value="$(find pcl_tools)/doc/examples/save/test.ply" />
        <param name="spin_once" type="bool" value="true" />
        <param name="point_cloud_frame_id" type="string" value="/camera" />
        <param name="include_tf" type="bool" value="true" />
            <param name="tf_filepath" type="string" value="$(find pcl_tools)/doc/examples/save/test_tf.pose" />
            <param name="world_frame_id" type="string" value="/world" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pcl_tools)/rviz/save_load_point_cloud.rviz" />
</launch>
